pages/12.Plugins/20.Zapier/docs.en.md

```diff
@@ -17,21 +17,32 @@ This integration integrates any Mautic instance with 1.100+ third party web serv
 
 ## Requirements
 
-1. Mautic version 2.9.0 or newer with SSL - basic authentication requires HTTPS to be secure.
-3. Zapier account. Free or Paid. [View Zapier pricing][zapier-pricing].
+Zapier account. Free or Paid. [View Zapier pricing][zapier-pricing].
 
 ## Installation
 
-There is no need to install any additional plugin to your Mautic instance. Zapier works with the API and webhooks already available within Mautic. This Zapier integration use Basic Authentication to access Mautic API. It is disabled by default so enable this first.
+There is no need to install any additional plugin. Zapier works with the API and webhooks already available within Mautic.
+
+### Set up OAuth2 Authentication in Mautic for Zapier 3.x
+
+This new version of the app improves authentication by implementing the OAuth2 protocol
 
 1. Open the admin menu
-2. Go to Mautic's global configuration
-3. Go to API Settings
-4. Set __API enabled?__ to __Yes__.
-5. Set __Enable HTTP basic auth?__ to __Yes__.
-6. Save the configuration.
+2. Go to Mautic's global configuration > API Settings
+3. Set __API enabled?__ to __Yes__.
+4. Save the configuration.
+5. Go to Mautic's global configuration > API Credentials > __OAuth2__
+6. Create new __OAuth2___ application with redirect URI: __example.com__
```

Comment:
Where do they get the redirect URL - or is this the URL of their Mautic instance?

Reply:
It's Zapier giving it.
Example : https://zapier.com/dashboard/auth/oauth/return/App80678CLIAPI/

Reply:
Ah, might be good to make that clear (eg redirect URL from Zapier)?